packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -1593,7 +1594,9 @@ def send_raw_transaction(
             f"Transaction sent! Received transaction digest: {tx_hash}"
         )
 
-        if tx_hash != tx_hash_backup:
+        # the check whether its solana is temporary until we have issues in the plugin
+        # fixed https://github.com/valory-xyz/open-aea/pull/708#discussion_r1448366974
+        if tx_hash != tx_hash_backup and chain_id != SOLANA_LEDGER_ID:
```

Comment:
this should not be the case anymore, that PR is merged @angrybayblade

Reply:
https://github.com/valory-xyz/open-autonomy/pull/2139/commits/599f2fd524d1d65e38a2c1e6abc52f68dfb27f85